tensorflow/lite/micro/tools/make/Makefile

```diff
@@ -255,18 +260,19 @@ $(patsubst %.S,%.o,$(patsubst %.cc,%.o,$(patsubst %.c,%.o,$(THIRD_PARTY_CC_SRCS)
 
 # For normal manually-created TensorFlow C++ source files.
 $(OBJDIR)%.o: %.cc $(THIRD_PARTY_TARGETS)
+	echo $(THIRD_PARTY_TARGETS)
 	@mkdir -p $(dir $@)
-	$(CXX) $(CXXFLAGS) $(INCLUDES) -c $< -o $@
+	$(CXX) $(CXXFLAGS) $(INCLUDES) -fPIC -c $< -o $@
```

Comment:
is -fPIC mandatory?

Reply:
I'm unable to compile tensorflow-litemicro.a file into a shared object library without -fPIC, however, If that is not something that needs to be supported I can take it out.

Reply:
I'm a bit confused, I thought .so is a shared library while .a (applies to us) is statically linked in.

Regardless from documentation, it looks that -fPIC is desirable. I'm OK with leaving it as it is.

Reply:
I think we should remove -fPIC here, and add it back in to particular platforms that need it, since in my experience it hasn't been very portable.